notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb

```diff
@@ -481,7 +450,7 @@
     "tags": []
    },
    "source": [
-    "Create a quantized model from a pre-trained `FP32` model and configuration object created with the dictionary defined in the previous cell and a DataLoader. See the [NNCF documentation](https://github.com/openvinotoolkit/nncf/blob/develop/docs/Usage.md) for more information."
+    "Create a quantized model from the pre-trained `FP32` model and the calibration dataset."
```

Comment:
could you please provide more explanation for this part?

Reply:
More description was added.